ProjectGroupA/src/main/java/com/fundation/search/controller/Controller.java

```diff
@@ -0,0 +1,86 @@
+/*******************************************************************************
+ * @(#)AssetFile.java Copyright (c) 2018 Jalasoft.
+ *  2643 Av Melchor Perez de Olguin, Colquiri Sud, Cochabamba, Bolivia.
+ *  All rights reserved.
+ *  <p>
+ *  This software is the confidential and proprietary information of
+ *  Jalasoft, ("Confidential Information").  You shall not
+ *  disclose such Confidential Information and shall use it only in
+ *  accordance with the terms of the license agreement you entered into
+ *  with Jalasoft.
+ *******************************************************************************/
+package com.fundation.search.controller;
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+/**
+ * This class is going to interact with the view and model class.
+ *
+ * @author Jacqueline Rosales
+ * @version 1.0.
+ */
+public class Controller {	
+	/**  
+    	searcher is a object of type Searcher class, Searcher class must be implemented in model package.
+    	val is a object of type Validator class.
+    	file is object of type SearcherFile class where the data related to one file will be stored and managed as object by model, controller and view packages.
+    	interface is object of type SearcherGUI class where UI should be implemented.*/
+	//private Searcher searcher;
+	private Validator val;
+	private SearcherFile file;
+	//private SearcherGUI interface;
+	
+	/** 
+     *  constructor for Controller class where the objects declared above will be  initialized by its respective constructors.*/
+	public Controller(){
+		//interface= new SearcherGUI();
+		//searcher= new Searcher();
+		val= new Validator();
+		file = new SearcherFile();
+	}
+	
+	/**
+     *  method to call the search method.*/
+	public void init() {
+		search();
+	}
+	/**
+     *  method where the values to path, filename and extension will be assigned to file object and all them will be validated,
+     *  then will be sent to model to start with search process and then showed under UI. */
+	private void search() {
+		try {
+			file.path= getPath();  
+			file.fileName= getFileName();
+			file.ext= getExtension();
+			val.validate(file);
+			//searcher.search(file);
+			displayResults();
+			}catch (Exception ex) {	System.out.println(ex.getMessage()); }
```

Comment:
It seems we need to change the format (Need Paolo inputs) to 
} catch (....) {
        Sys....;
}